### PR TITLE
change_delegated_auth_user

### PR DESCRIPTION
**Pull request by dawit-simeon**

### CODE REVIEW

File: how-to.md [hunk 1]
```diff
@@ -121,3 +121,7 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
   * **Tenants** > **Repos** > select dropdown at the top of the page > **Manage repositories** > **...** (next to the tenant you want to remove) > **Delete** > follow the on-screen instructions to delete the repository
   * **Pipelines** > **...** (next to the tenant you want to remove) > follow the on-screen instructions to delete the pipeline
   * **Environments** (on the left pane under **Pipelines**) > select the tenant you want to remove > **...** > **Delete** > **Delete**
+
+## Change user used for delegated authentication
+
+* To change the user you are using for delegated auth, navigate to [Simeon](https://app.simeoncloud.com/install) > toggle **off** New Tenant > select the tenant under Display name > INSTALL > follow the steps on the screen > SYNC NOW > when the status of the tenant Sync is **Pending Authentication** authenticate with a different user.
```
Comment (david-simeon) [inline how-to.md:127]: spell it out for consistency - delegated authentication

not delegated auth

Comment (david-simeon) [inline how-to.md:127]: We use bold in docs a button on a screen, not for emphasis. Unbold **off** and bold New Tenant

Comma after **pending authentication** and before authenticate

Comment (dawit-simeon) [reply how-to.md]: resolved

Comment (dawit-simeon) [reply how-to.md]: resolved

File: how-to.md [hunk 2]
```diff
@@ -121,3 +121,7 @@ You can verify the licenses in your tenant [in the Azure Portal](https://portal.
   * **Tenants** > **Repos** > select dropdown at the top of the page > **Manage repositories** > **...** (next to the tenant you want to remove) > **Delete** > follow the on-screen instructions to delete the repository
   * **Pipelines** > **...** (next to the tenant you want to remove) > follow the on-screen instructions to delete the pipeline
   * **Environments** (on the left pane under **Pipelines**) > select the tenant you want to remove > **...** > **Delete** > **Delete**
+
+## Change user used for delegated authentication
+
+* To change the user you are using for delegated authentication, navigate to [Simeon](https://app.simeoncloud.com/install) > toggle off **New Tenant** > select the tenant under Display name > INSTALL > follow the steps on the screen > SYNC NOW > when the status of the tenant Sync is **Pending Authentication**, authenticate with a different user.
```
Comment (david-simeon) [inline how-to.md:127]: For consistency with the other how tos, get rid of "to change the user you are using for delegated authentication"

Comment (dawit-simeon) [reply how-to.md]: resolved

Comment (david-simeon) [inline how-to.md]: We should probably add this sentence at the end here "If you don't authenticate within 5 minutes after the status changes to *Pending Authentication*, the Sync will time out."

Comment (dawit-simeon) [reply how-to.md]: resolved

**Review by david-simeon — APPROVED** [review]

**Review by rdorta-simeon — APPROVED** [review]

**Pull request merged**